src/sdk/model/advanced-backups/backup-group/__json__/iaas-backup-subscription-json.ts

```diff
@@ -0,0 +1,17 @@
+import { CurrencyCode } from '../../../common/billing/__json__/currency-code-type';
+
+/**
+ * IaaS backup subscription JSON.
+ */
+export interface IaasBackupSubscriptionJson {
+  company_id: string;
+  location_id: string;
+  org_uuid: string;
+  vdc_uuid: string;
+  activation_time: number;
+  deactivation_time?: number;
+  reserved_amount_gb: number;
+  reserved_cost: number;
+  burst_cost_per_gbhour: number;
```

Comment:
It should be: 'burst_cost_per_gb_hour'. If it is not the case API side, then they have a JSON formatting issue on their side ;-).

Reply:
it looks like this does match the [API response](http://doc.10.api.iland.test/1.0/#/vDCs/listIaasBackupSubscriptions)

Reply:
In this case, should we A) request and wait for an API update or B) merge and update the SDK with separate patch later.

Reply:
Ok, so this is an issue API side. When it's happening, we need to inform the platform people (i guess in this case it's Cory ? 🤔 ) that they have a JSON format issue. This is a really common issue, it happens. We just need to inform them during the development when we see stuff like that.
In this particular case, i would say, lets create a ticket platform side to ask for the change, and assign it to Cory with a 'Major' priority then create another one SDK side to update it in a future patch (once the fix is deployed platform side) .
That way we can merge this one so @iamtash can PR upon ab-beta branch directly. 😄

Reply:
Sounds good. I'll create an issue for the API platform fix.

Reply:
JSON formatting fix was [merged](http://git.iland.test/projects/IC/repos/iland-core/pull-requests/4394/overview)